<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>610</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Almespar ORF Finder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black','Arial'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial Black','Arial Black'; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>631</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #309010; 
alternate-background-color: #402030;
selection-background-color:#282828;
 }</string>
    </property>
    <column>
     <property name="text">
      <string>Lable</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Strand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Frame</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stop</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Length</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black','Arial'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial Black'; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>611</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #51255251; 
alternate-background-color: #606060;
selection-background-color:#282828;
 }</string>
    </property>
    <column>
     <property name="text">
      <string>Lable</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Strand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Frame</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stop</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Length</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Complement  Sequence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nucleotide Sequence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Minimal ORF length (nt):</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Genetic code:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>530</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1. The Standard Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2. The Vertebrate Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3. The Yeast Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4. The Mold, Protozoan, and Coelenterate Mitochondrial Code and the Mycoplasma/Spiroplasma Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5. The Invertebrate Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6. The Ciliate, Dasycladacean and Hexamita Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9. The Echinoderm and Flatworm Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10. The Euplotid Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11. The Bacterial, Archaeal and Plant Plastid Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12. The Alternative Yeast Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13. The Ascidian Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14. The Alternative Flatworm Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15.Blepharisma Nuclear</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16. Chlorophycean Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21. Trematode Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22. Scenedesmus obliquus Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23. Thraustochytrium Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24. Rhabdopleuridae Mitochondrial Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25. Candidate Division SR1 and Gracilibacteria Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26. Pachysolen tannophilus Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27. Karyorelict Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28. Condylostoma Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29. Mesodinium Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30. Peritrich Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31. Blastocrithidia Nuclear Code</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>33. Cephalodiscidae Mitochondrial UAA-Tyr Code</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Forward ORFs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reverse ORFs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Read FASTA</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>381</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> &quot;ATG&quot; and alternative initiation codons</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="mycrs.qrc">:/69756494_2446786435377785_4710438350333411328_n.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>1141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="mycrs.qrc">:/2022-01-15_18-25-28.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Print to PDF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="mycrs.qrc"/>
 </resources>
 <connections/>
</ui>
